README.md

```diff
@@ -157,6 +157,7 @@ You can pass the following options to `cmake ..`.
 * `-DLINK_PROFILER` -- Link with gperftools (available profiler options are listed [here](https://github.com/gperftools/gperftools))
 * `-DKVS_STACK_SIZE` -- Default stack size for threads created using THREAD_CREATE()
 * `-DPKG_CONFIG_EXECUTABLE` -- Set pkg config path. This might be required to find gstreamer's pkg config specifically on Windows.
+* `-DENABLE_GST_SAMPLE_RECEIVER` -- Enable gstreamer receiver to receive media while running the Viewer and GstMaster samples
```

Comment:
We should also add what gstreamer is needed on your system, and what command to install it.